_capabilities/utdallas-ufl-xvs.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Explainable Video Activity Search Tool"
+excerpt: "Demo of our explainable query-building tool to search activities within a collection of videos."
+tags: 
+- Analytics
```

Comment:
Spaces for this contribution will also need to be fixed to respect the hierarchical nature of yaml formatting.